include/flamegpu/runtime/detail/curve/DeviceCurve.cuh

```diff
@@ -231,6 +233,69 @@ class DeviceCurve {
     template <typename T, unsigned int N = 0, unsigned int M>
     __device__ __forceinline__ static T getEnvironmentArrayProperty(const char(&propertyName)[M], unsigned int array_index);
 
+
+    /**
+     * Retrieve the address of the specified environment directed graph's pbm (CSR) for edges outgoing from each vertex
+     *
+     * @param graphHash The hash of the affected graph
+     * @return The requested address
+     * @throws exception::DeviceError (Only when SEATBELTS==ON) If the specified graph is not found in the cuRVE hashtable, or it's details are invalid.
+     */
+    __device__ __forceinline__ static unsigned int *getEnvironmentDirectedGraphPBM(VariableHash graphHash);
```

Comment:
I'd prefer CSR/CSC in the names not PBM / IPBM, but prolly not worth the effort to fix / just my bias from doing more graph stuff.

Reply:
Doesn't bother me if you wish to refactor, so long as "PBM" is in the doc comment for my comprehension.